modules/features/falcon_feature_content_blocks/config/install/crop.type.hero_large.yml

```diff
@@ -0,0 +1,11 @@
+langcode: en
+status: true
+dependencies: {  }
+label: 'Hero Large'
```

Comment:
What I would suggest doing for all image styles, is to add the resulting resolution to the label (not to the machine name!). So that the machine name stays `hero_large` forever, but the label is `Hero Large (1600x500)` or `Hero Large (w1600)` for example. Really handy way to handle image style labels.